File: java/program/Product.java
// deklarasi class
class Product extends Shirt {

    // attibut private bertipe data string
    private String idProduct;
    private String nama;
    private String brand;
    private String price;

    // constructor tanpa parameter
    public Product() {
        this.idProduct = "";
        this.nama = "";
        this.brand = "";
        this.price = "";

    }

    // constructor dengan parameter idProduct, nama, brand
    public Product(String idProduct, String nama, String brand, String price) {
        this.idProduct = idProduct;
        this.nama = nama;
        this.brand = brand;
        this.price = price;

    }

    // setter dan getter untuk setiap atribut idProduct, nama, brand, dan price
    public String get_idProduct() {
        return this.idProduct;
    }

    public void set_idProduct(String idProduct) {
        this.idProduct = idProduct;
    }

    public String get_nama() {
        return this.nama;
    }

    public void set_nama(String nama) {
        this.nama = nama;
    }

    public String get_brand() {
        return this.brand;
    }

    public void set_brand(String brand) {
        this.brand = brand;
    }

    public String get_price() {
        return this.price;
    }

    public void set_price(String price) {
        this.price = price;
    }

};

File: python/program/Product.py
class Product:

    # deklarasi private atribut
    __idProduct = ""
    __nama = ""
    __brand = ""
    __price = ""

    # constructor
    def __init__(self, idProduct="", nama="", brand="", price=""):
        self.__idProduct = idProduct
        self.__nama = nama
        self.__brand = brand
        self.__price = price

    # setter dan getter
    def get_idProduct(self):
        return self.__idProduct

    def set_idProduct(self, idProduct):
        self.__idProduct = idProduct

    def get_nama(self):
        return self.__nama

    def set_nama(self, nama):
        self.__nama = nama

    def get_brand(self):
        return self.__brand

    def set_brand(self, brand):
        self.__brand = brand

    def get_price(self):
        return self.__price

    def set_price(self, price):
        self.__price = price
